### PR TITLE
Add C# language version

### PR DESCRIPTION
**Pull request by jaredpar**
Include the C# language version in the release.json file.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @jcouv another place where we track the C# version.

File: release-notes/releases.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
         "checksums-sdk": "https://dotnetcli.blob.core.windows.net/dotnet/checksums/2.1.300-preview2-008533-sdk-sha.txt",
         "security": "FALSE",
         "lts-runtime": "FALSE",
-        "lts-sdk": "FALSE"
+        "lts-sdk": "FALSE",
+        "csharp-language": "7.3"
```
Comment (jcouv) [inline release-notes/releases.json]: Thanks for the heads-up.
Is `csharp-language` a well-known string? Should it be `csharplang`?
Should we also record VB language version?

I assume the .NET Core folks would be responsible for updating this when they pick up new versions of the compiler packages. Is that right?

Comment (jaredpar) [reply release-notes/releases.json]: This is still hand authored and i just found out about it today. Im' trying to back fill all of the versions that I know of. 

The pattern seemed to be using `-` to separate words in this file hence I followed that for `csharp-language`.

Comment (jcouv) [reply release-notes/releases.json]: Who owns updating this?

Comment (jaredpar) [reply release-notes/releases.json]: Working that out.

Comment (jcouv) [reply release-notes/releases.json]: @jaredpar Did we resolve the question of ownership? Thanks

**Comment by jcouv** [comment]
@jaredpar Did we sort out ownership for this moving forward?
When it comes to 8.0, are we going to update this file in 8.0 beta timeframe, or wait until 8.0?

**Pull request merged**